porcupine/plugins/directory_tree.py

```diff
@@ -216,15 +224,13 @@ def add_project(self, root_path: Path, *, refresh: bool = True) -> None:
                 self.move(project_item_id, "", 0)  # type: ignore[no-untyped-call]
                 return
 
-        # TODO: show long paths more nicely
-        text = str(root_path)
-        if Path.home() in root_path.parents:
-            text = text.replace(str(Path.home()), "~", 1)
-
-        # Add project to beginning so it won't be hidden soon
         self._project_num_counter += 1
         project_item_id = self.insert(
-            "", 0, f"project:{self._project_num_counter}:{root_path}", text=text, open=False
+            "",
+            0,  # beginning, so it won't be hidden soon
+            f"project:{self._project_num_counter}:{root_path}",
```

Comment:
It's confusing how this is here but it ends up being the `project_item_id`, and there's also a loop with `project_item_id`